pilot/pkg/serviceregistry/kube/controller/controller_test.go

```diff
@@ -833,7 +833,7 @@ func TestGetProxyServiceTargets_WorkloadInstance(t *testing.T) {
 		Namespace: "bookinfo-ratings",
 		Endpoint: &model.IstioEndpoint{
 			Labels:       labels.Instance{"app": "ratings"},
-			Address:      "2.2.2.21",
```

Comment:
TestGetProxyServiceTargets_WorkloadInstance() requires dual-stack tests

Reply:
done!  Add dual stack tests for `ratings-1` and `productpage-1`